central/scannerdefinitions/handler/handler.go

```diff
@@ -452,59 +455,84 @@ func (h *httpHandler) openMostRecentDefinitions(ctx context.Context, uuid string
 	return
 }
 
-func (h *httpHandler) openMostRecentV4File(ctx context.Context, t updaterType, updaterKey, fileName string) (file *vulDefFile, err error) {
+// v4OpenOpts are options to open most recent V4 definition files.
+type v4OpenOpts struct {
+	// urlPath specifies the update URL path when setting up online updaters.
+	urlPath string
+	// mappingFile specifies the mapping file to open for mappings updaters.
+	mappingFile string
```

Comment:
Is the inclusion of both `vulnVersion` and `vulnBundle` in the options due to the separation of the vulnerability bundle into multiple-bundles?
If that's not the case I am a bit confused with having `mappingFile`, `vulnVersion` and `vulnBundle` in the Options. Because in one single request, the user can only specify one type of file (either mapping file or vulnd bundle). Keep using `fileName` should do the job.

Reply:
> Is the inclusion of both vulnVersion and vulnBundle in the options due to the separation of the vulnerability bundle into multiple-bundles?

Not really. The `vulnBundle` and `vulnVersion` is used in online and offline opening steps. But it will help, since the next PR for the multi-bundle will simply be changing the bundle name to something else.

Reply:
> If that's not the case I am a bit confused with having mappingFile, vulnVersion and vulnBundle in the Options. Because in one single request, the user can only specify one type of file (either mapping file or vulnd bundle). 

The options makes it explicit the purpose of the data passed in the parameters. Before, we were resuing parameters like `updaterKey` and `fileName` to mean different things depending on which updater type was selected. Now we have specific variables for each type of data. The updater type determines how the data will be used. Some of the options are not supposed to be used in all updater types.

> Keep using fileName should do the job.

Oh yes. This is not a change to make the job happen. It was happening before. This is a change to improve the overall maintainability of the definition handlers.

Reply:
Feel free to resolve this if you don't have any more questions.